server/middlewares/frontendMiddleware.js

```diff
@@ -87,6 +129,98 @@ const addDevMiddlewares = (app, webpackConfig) => {
     res.send('loaderio-446030d79af6fc10143acfa9b2f0613f');
   });
 
+  // reserveSsrRoutes(app);
+  app.get(/[0-9]+-*/, async (req, res) => {
+    rp({
+      uri: 'https://api.studykik.com/api/v1/landingPages/2835078/fetchLanding',
```

Comment:
what is that for?

does it work for only url `https://api.studykik.com/api/v1/landingPages/2835078/fetchLanding`?